adam-core/src/main/scala/org/bdgenomics/adam/models/Interval.scala

```diff
@@ -41,4 +42,19 @@ trait Interval {
    */
   def width: Long = end - start
 
+  /**
+   * Need to implement getStart function from Locatable. (1-based start position. closed)
```

Comment:
I don't think we can hide it, but we could mark it as `@deprecated` to hint that it is only for use with legacy codebases?